namui/src/namui/manager/common/typeface_manager.rs

```diff
@@ -1,26 +1,25 @@
+use dashmap::DashMap;
+
 use crate::namui::{self, Typeface, TypefaceType};
-use std::collections::HashMap;
 
 pub struct TypefaceManager {
-    typefaces: HashMap<TypefaceType, Box<Typeface>>,
+    typefaces: DashMap<TypefaceType, Typeface>,
```

Comment:
To support immutable reference with manager, I replaced HashMap with DashMap which support lock-free map with immutable reference.
If you want to put String inside of manager struct, you can use Mutex.
```rs
pub struct YourManager {
    id: Mutex<String> // << like this
}
```